server/views/components/input/input.njk

```diff
@@ -1,6 +1,6 @@
 <label class="input__label flex flex--v-center" for="{{ model.id }}">
   <input id="{{ model.id }}"
-    class="input input--{{ model.type }} {{ data.fontStyles or {s:'HNL3', m:'HNL2'} | fontClasses }} js-input"
+    class="input input--{{ model.type }} {% if data.fontStyles %} {{data.fontStyles | fontClasses}} {% else %} {{ {s:'HNL3', m:'HNL2'} | fontClasses }} {% endif %} js-input"
```

Comment:
is this doing the same thing as it was before?

Reply:
same thing as it was meant to be doing, but we were just getting [object object] instead of the actual classes